### PR TITLE
docs: add inference section

### PR DESCRIPTION
**Pull request by bwanglzu**
Improve three examples with `inference` section: https://ft-docs-improve-examples--jina-docs.netlify.app/docs-improve-examples/tasks/text-to-text/#inference

---

- [x] This PR references an open issue
- [x] I have added a line about this change to CHANGELOG

### CODE REVIEW

File: docs/tasks/text-to-image.md [hunk 1]
```diff
@@ -119,6 +119,34 @@ We have done the evaulation for you in the table below.
...
+:class: hint
+When fine-tuning CLIP, we are fine-tuning the CLIPVisionEncoder and CLIPTextEncoder in parallel.
+The artifact contains two models: `clip-vision` and `clip-text`.
+The parameter `select_model` tells finetune which model to use for inference, in the above example,
+we use `clip-text` to encode a Document with text.
+```
+
+Check out [clip-as-service](https://clip-as-service.jina.ai/user-guides/finetuner/?highlight=finetuner#fine-tune-models) to know how to plug-in fine-tuned model there.
+
+That's it! If you want to integrate fine-tuned model into your Jina Flow, please check out {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
```
Comment (gmastrapas) [inline docs/tasks/text-to-image.md:152]: integrate the

File: docs/tasks/text-to-image.md [hunk 2]
```diff
@@ -119,6 +119,34 @@ We have done the evaulation for you in the table below.
...
+finetuner.encode(model=model, data=test_da)
+print(test_da.embeddings)
+```
+
+```{admonition} what is select_model?
+:class: hint
+When fine-tuning CLIP, we are fine-tuning the CLIPVisionEncoder and CLIPTextEncoder in parallel.
+The artifact contains two models: `clip-vision` and `clip-text`.
+The parameter `select_model` tells finetune which model to use for inference, in the above example,
+we use `clip-text` to encode a Document with text.
```
Comment (gmastrapas) [inline docs/tasks/text-to-image.md:147]: with text content

File: docs/tasks/text-to-image.md [hunk 3]
```diff
@@ -119,6 +119,34 @@ We have done the evaulation for you in the table below.
...
-run.save_artifact('clip-model')
+artifact = run.save_artifact('clip-model')
 ```
-That's it! Now you have a fine-tuned model which is ready to be {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
\ No newline at end of file
+
+## Inference
+
+Now you saved the `artifact` into your host machine,
+let's use fine-tuned model to encode a new `Document`.
```
Comment (gmastrapas) [inline docs/tasks/text-to-image.md:128]: use the fine-tuned model

File: docs/tasks/image-to-image.md [hunk 4]
```diff
@@ -129,6 +129,24 @@ What you can do for now is to call {meth}`~finetuner.run.Run.logs()` in the end
...
-run.save_artifact('resnet-model')
+artifact = run.save_artifact('resnet-model')
 ```
-That's it! Now you have a fine-tuned model which is ready to be {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
\ No newline at end of file
+
+## Inference
+
+Now you saved the `artifact` into your host machine,
+let's use fine-tuned model to encode a new `Document`.
```
Comment (gmastrapas) [inline docs/tasks/image-to-image.md:138]: use the

File: docs/tasks/image-to-image.md [hunk 5]
```diff
@@ -129,6 +129,24 @@ What you can do for now is to call {meth}`~finetuner.run.Run.logs()` in the end
...
+# Load model from artifact
+model = finetuner.get_model(artifact=artifact)
+# Prepare some text to encode, change the placeholder image uri to an image on your machine
+test_da = DocumentArray([Document(uri='my-image.png')])
+# Encoding will happen in-place in your `DocumentArray`
+finetuner.encode(model=model, data=test_da)
+print(test_da.embeddings)
+```
+
+That's it! If you want to integrate fine-tuned model into your Jina Flow, please check out {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
```
Comment (guenthermi) [inline docs/tasks/image-to-image.md:152]:
```suggestion
That's it! If you want to integrate a fine-tuned model into your Jina Flow, please check out {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
```

File: docs/tasks/text-to-image.md (hunk 3 above)
Comment (guenthermi) [inline docs/tasks/text-to-image.md:128]:
```suggestion
let's use the fine-tuned model to encode a new `Document`.
```

File: docs/tasks/text-to-image.md [hunk 6]
```diff
@@ -119,6 +119,34 @@ We have done the evaulation for you in the table below.
...
+
+```{admonition} what is select_model?
+:class: hint
+When fine-tuning CLIP, we are fine-tuning the CLIPVisionEncoder and CLIPTextEncoder in parallel.
+The artifact contains two models: `clip-vision` and `clip-text`.
+The parameter `select_model` tells finetune which model to use for inference, in the above example,
+we use `clip-text` to encode a Document with text.
+```
+
+Check out [clip-as-service](https://clip-as-service.jina.ai/user-guides/finetuner/?highlight=finetuner#fine-tune-models) to know how to plug-in fine-tuned model there.
```
Comment (guenthermi) [inline docs/tasks/text-to-image.md:150]:
```suggestion
Check out [clip-as-service](https://clip-as-service.jina.ai/user-guides/finetuner/?highlight=finetuner#fine-tune-models) to know how to plug-in fine-tuned models there.
```

File: docs/tasks/text-to-text.md [hunk 7]
```diff
@@ -196,6 +196,24 @@ print(f'Run logs: {run.logs()}')
...
 ```python
-run.save_artifact('bert-model')
+artifact = run.save_artifact('bert-model')
 ```
-That's it! Now you have a fine-tuned model which is ready to be {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
+
+## Inference
+
+Now you saved the `artifact` into your host machine,
+let's use fine-tuned model to encode a new `Document`.
```
Comment (guenthermi) [inline docs/tasks/text-to-text.md:205]:
```suggestion
let's use the fine-tuned model to encode a new `Document`.
```

File: docs/tasks/text-to-text.md [hunk 8]
```diff
@@ -196,6 +196,24 @@ print(f'Run logs: {run.logs()}')
...
+# Load model from artifact
+model = finetuner.get_model(artifact=artifact)
+# Prepare some text to encode
+test_da = DocumentArray([Document(text='some text to encode')])
+# Encoding will happen in-place in your `DocumentArray`
+finetuner.encode(model=model, data=test_da)
+print(test_da.embeddings)
+```
+
+That's it! If you want to integrate fine-tuned model into your Jina Flow, please check out {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
```
Comment (guenthermi) [inline docs/tasks/text-to-text.md:219]:
```suggestion
That's it! If you want to integrate a fine-tuned model into your Jina Flow, please check out {ref}`integrated with the Jina ecosystem <integrate-with-jina>`.
```

**Review by guenthermi — APPROVED** [review]
added some minor grammatical suggestions

**Review by gmastrapas — APPROVED** [review]

**Pull request merged**